src/masternodes/mn_rpc.cpp

```diff
@@ -286,8 +286,8 @@ UniValue resignmasternode(const JSONRPCRequest& request) {
                        "\"hash\"                  (string) The hex-encoded hash of broadcasted transaction\n"
                },
                RPCExamples{
-                   HelpExampleCli("resignmasternode", "mn_id \"[{\\\"txid\\\":\\\"id\\\",\\\"vout\\\":0}]\"")
-                   + HelpExampleRpc("resignmasternode", "mn_id \"[{\\\"txid\\\":\\\"id\\\",\\\"vout\\\":0}]\"")
+                   HelpExampleCli("resignmasternode", "mn_id \"[{\"txid\":\"id\",\"vout\":0}]\"")
```

Comment:
I think this needs the single quotes added

Reply:
Yes, the quote need to keep, because parameters need to parse as JSON, otherwise the example cannot run